BLUESPAWN-win-client/src/user/BLUESPAWN.cpp

```diff
@@ -0,0 +1,459 @@
+    mitigationRecord.RegisterMitigation(std::make_shared<Mitigations::MitigateM1028WFW>());
+    mitigationRecord.RegisterMitigation(std::make_shared<Mitigations::MitigateM1035RDP>());
+    mitigationRecord.RegisterMitigation(std::make_shared<Mitigations::MitigateM1042LLMNR>());
+    mitigationRecord.RegisterMitigation(std::make_shared<Mitigations::MitigateM1042NBT>());
+    mitigationRecord.RegisterMitigation(std::make_shared<Mitigations::MitigateM1042WSH>());
+    mitigationRecord.RegisterMitigation(std::make_shared<Mitigations::MitigateM1047>());
+    mitigationRecord.RegisterMitigation(std::make_shared<Mitigations::MitigateM1054RDP>());
+    mitigationRecord.RegisterMitigation(std::make_shared<Mitigations::MitigateM1054WSC>());
+    mitigationRecord.RegisterMitigation(std::make_shared<Mitigations::MitigateV1093>());
+    mitigationRecord.RegisterMitigation(std::make_shared<Mitigations::MitigateV1153>());
+    mitigationRecord.RegisterMitigation(std::make_shared<Mitigations::MitigateV3338>());
+    mitigationRecord.RegisterMitigation(std::make_shared<Mitigations::MitigateV3340>());
+    mitigationRecord.RegisterMitigation(std::make_shared<Mitigations::MitigateV3344>());
+    mitigationRecord.RegisterMitigation(std::make_shared<Mitigations::MitigateV3379>());
+    mitigationRecord.RegisterMitigation(std::make_shared<Mitigations::MitigateV3479>());
+    mitigationRecord.RegisterMitigation(std::make_shared<Mitigations::MitigateV63597>());
+    mitigationRecord.RegisterMitigation(std::make_shared<Mitigations::MitigateV63687>());
+    mitigationRecord.RegisterMitigation(std::make_shared<Mitigations::MitigateV63753>());
+    mitigationRecord.RegisterMitigation(std::make_shared<Mitigations::MitigateV63817>());
+    mitigationRecord.RegisterMitigation(std::make_shared<Mitigations::MitigateV63825>());
+    mitigationRecord.RegisterMitigation(std::make_shared<Mitigations::MitigateV63829>());
+    mitigationRecord.RegisterMitigation(std::make_shared<Mitigations::MitigateV71769>());
+    mitigationRecord.RegisterMitigation(std::make_shared<Mitigations::MitigateV72753>());
+    mitigationRecord.RegisterMitigation(std::make_shared<Mitigations::MitigateV73511>());
+    mitigationRecord.RegisterMitigation(std::make_shared<Mitigations::MitigateV73519>());
+    mitigationRecord.RegisterMitigation(std::make_shared<Mitigations::MitigateV73585>());
+
+    reactions.emplace("carve-memory", std::make_unique<Reactions::CarveMemoryReaction>());
+    reactions.emplace("delete-file", std::make_unique<Reactions::DeleteFileReaction>());
+    reactions.emplace("quarantine-file", std::make_unique<Reactions::QuarantineFileReaction>());
+    reactions.emplace("remove-value", std::make_unique<Reactions::RemoveValueReaction>());
+    reactions.emplace("suspend", std::make_unique<Reactions::SuspendProcessReaction>());
+}
+
+void Bluespawn::RunHunts() {
+    Bluespawn::io.InformUser(L"Starting a Hunt");
+    DWORD tactics = UINT_MAX;
+    DWORD dataSources = UINT_MAX;
```

Comment:
Codacy found an issue: [Variable 'dataSources' is assigned a value that is never used.](https://app.codacy.com/manual/ION28/BLUESPAWN/pullRequest?prid=5982821)